docs/analyzer/user_guide.md

```diff
@@ -682,8 +695,9 @@ to collect these so the analysis process can run in the same environment as the
 original build. However, there are some GCC-specific locations (usually with
 name `include-fixed`) which may be incompatible with other compilers and may
 cause failure in analysis. CodeChecker omits these GCC-specific paths from the
-analysis unless `--keep-gcc-include-fixed` flag is given. For further
-information see [GCC incompatibilities](gcc_incompatibilities.md).
+analysis unless `--keep-gcc-include-fixed` or `--keep-gcc-intrin` flag is
+given. For further information see
+[GCC incompatibilities](gcc_incompatibilities.md).
```

Comment:
Please add the `--keep-gcc-intrin` flag usage to the `GCC incompatibilities` section with the description when should the user enable this flag.